src/vulkan-renderer/wrapper/cpu_texture.cpp

```diff
@@ -0,0 +1,87 @@
+﻿#include "inexor/vulkan-renderer/wrapper/cpu_texture.hpp"
+
+#define STB_IMAGE_IMPLEMENTATION
+#include <spdlog/spdlog.h>
+#include <stb_image.h>
+
+#include <array>
+#include <vector>
+
+namespace inexor::vulkan_renderer::wrapper {
+
+CpuTexture::CpuTexture() : m_file_name("default texture"), m_name("default texture") {
+    generate_error_texture_data();
+}
+
+CpuTexture::CpuTexture(const std::string &file_name, const std::string &name) : m_file_name(file_name), m_name(name) {
```

Comment:
I changed the name variable to value / move.  However, I kept the file_name as const / ref because we are no longer storing it.  Should it also be by value?

Reply:
Should be const ref here then.

Reply:
Sounds good.  It should be fixed now.